onnxscript/rewriter/ort_fusions/sdpa_test.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SDPA fusion test cases."""
+
+from __future__ import annotations
+
+import math
+import unittest
+
+import numpy
+
+import onnxscript.ir as ir
+import onnxscript.optimizer
+from onnxscript import script
+from onnxscript.onnx_opset import opset18 as op
+from onnxscript.onnx_types import FLOAT
+from onnxscript.rewriter.ort_fusions.sdpa import fuse_sdpa
+
+batchsize = 2
```

Comment:
```suggestion
BATCH_SIZE = 2
```
here and below: all caps for constants?

Reply:
I am tempted to use B, S, N, H ... which is, hopefully, not cryptic in this context (with expansion in comments). Looks better for the shapes down below anyway.